extension/resources/walkthrough/plots.md

```diff
@@ -1,14 +1,61 @@
 # Plots
 
-The extension will display the following sections for selected experiments:
+Select one or more experiments to visualize them in the
+[`Plots Dashboard`](command:onCommand:dvc.showPlots). This is the extension's
+equivalent of the `dvc plots show` and `dvc plots diff` commands.
 
-- Data Series - JSON, YAML, CSV or TSV files visualized using Vega pre-defined
-  or custom Vega-Lite templates.
-- Images - a table used to display image plots side by side.
-- Trends - linear plots based on data from the experiments table.
+💡 If you don't have any DVC
+[plots](https://dvc.org/doc/command-reference/plots) in the project, start
+writing data points into JSON, YAML, CSV or TSV or saving plots as images
+(`.png`, etc) (check the [DVCLive](https://dvc.org/doc/dvclive) helper library
+if you use Python):
 
-Use `DVC: Show Plots` from the Command Palette to open the view for a `DVC`
-repository.
+<p align="center">
+  <img src="images/plots-dump-with-open-file.png"
+       alt="Code to Dump a JSON Plot File" />
+  <img src="images/plots-dump-with-dvclive.png"
+       alt="Code to Dump a JSON Plot File with DVCLive" />
+  <img src="images/plots-dump-image.png"
+       alt="Code to Dump an Image Plot File" />
+</p>
 
-The view can be manipulated using the controls shown in the extension's view
-container.
+Use `DVC: Show Plots` from the
+[Command Palette](command:workbench.action.quickOpen?%22>DVC:%20Show%20Plots%22)
+to open up the plots dashboard. The extension will display the following
+sections for selected experiments, that correspond to the different
+[types of plots](https://dvc.org/doc/command-reference/plots#supported-file-formats)
+supported by DVC:
+
+<p align="center">
+  <img src="images/plots-data-series.png"
+       alt="Plots: Data Series" />
+</p>
+
+`Data Series`. JSON, YAML, CSV or TSV files visualized using the predefined
+(e.g. confusion matrix, linear) or custom (Vega-lite) templates:
```

Comment:
This confused me, consider moving title above picture instead of having below (especially with `:`). Should be 

```
`Data Series`. JSON, YAML, CSV or TSV files visualized using the predefined
(e.g. confusion matrix, linear) or custom (Vega-lite) templates:

<p align="center">
  <img src="images/plots-data-series.png"
       alt="Plots: Data Series" />
</p>
```

for each section

Reply:
Current experience is this 

![image](https://user-images.githubusercontent.com/37993418/173506567-e914d917-9f5b-4cd9-ad1d-88de9128d672.png)

Reply:
yes, I will remove the `:`, but have to keep the flow like this for now to make plots visible in the w/o scrolling the page. We can optimize it later.